core/ArchiveProcessor/Rules.php

```diff
@@ -330,7 +330,9 @@ public static function getSelectableDoneFlagValues($includeInvalidated = true, P
 
     public static function isForceArchivingSinglePlugin()
     {
-        if (!SettingsServer::isArchivePhpTriggered()) {
+        if (!SettingsServer::isArchivePhpTriggered()
+            || Loader::getArchivingDepth() > 1
```

Comment:
@diosmosis not sure yet I understand why `Loader::getArchivingDepth() > 1` means we aren't forcing a single archive?

Reply:
this method is more "are we forcing archiving because pluginOnly=1" rather than "are we forcing the creation of a single plugin archive". when archiving depth > 1, then we're archiving a child period. in this case we want to use what's available and not "force archiving even if there's something usable there".

so if someone activates MultiChannelConversionAttribution, and it invalidates a month in the past, then we see that `pluginOnly=1&plugin=MultiChannelConversionAttribution`, we know we need to force archiving for that period, even if there is a 'done' or 'done.MultiChannelConversionAttribution' archive there. for the weeks inside, however, we assume they are already archived by core:archive and we don't want this branch to force archiving of a new week.

hope that makes sense...